src/main/resources/openapi.json

```diff
@@ -753,6 +753,74 @@
                 }
             ]
         },
+        "/admin/topics/{topicname}": {
+            "post": {
+                "tags": [
+                    "Topics"
+                ],
+                "description": "Creates a topic with given topic name.",
+                "operationId": "createTopic",
+                "responses": {
+                    "200": {
+                        "description": "Topic metadata",
+                        "content": {
+                            "application/vnd.kafka.v2+json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/TopicMetadata"
+                                }
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "The specified topic was not found.",
```

Comment:
what does this mean when you are creating a topic? You are not looking for a topic which could be not found, or?